CHANGELOG.md

```diff
@@ -9,7 +9,7 @@
 
 ## 34.13.0
 
-* Set up individual component CSS loading, **Note:** this change is experimental, it will be used in the `frontend` application first, please do not implement this feature in other applications ([PR #3342](https://github.com/alphagov/govuk_publishing_components/pull/3342))
+* Set up individual component CSS loading, **Note:** this change is experimental, it will be used in the `frontend` application first, please do not implement this feature in other applications ([PR #3014](https://github.com/alphagov/govuk_publishing_components/pull/3014))
```

Comment:
While you're changing this, can you make the entry match the PR name?

Reply:
Done 👍

Reply:
Thanks!